addons/purchase_requisition/wizard/bid_line_qty.py

```diff
@@ -1,18 +1,20 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.osv import fields, osv
-import openerp.addons.decimal_precision as dp
+from odoo import api, fields, models
+import odoo.addons.decimal_precision as dp
 
-class bid_line_qty(osv.osv_memory):
+
+class BidLineQty(models.TransientModel):
     _name = "bid.line.qty"
     _description = "Change Bid line quantity"
-    _columns = {
-        'qty': fields.float('Quantity', digits_compute=dp.get_precision('Product Unit of Measure'), required=True),
-    }
 
-    def change_qty(self, cr, uid, ids, context=None):
-        active_ids = context and context.get('active_ids', [])
-        data = self.browse(cr, uid, ids, context=context)[0]
-        self.pool.get('purchase.order.line').write(cr, uid, active_ids, {'quantity_tendered': data.qty})
+    qty = fields.Float(string='Quantity', digits_compute=dp.get_precision('Product Unit of Measure'), required=True)
+
+    @api.multi
+    def change_qty(self):
+        self.ensure_one()
+        active_ids = self.env.context.get('active_ids', [])
+        Po_line = self.env['purchase.order.line'].browse(active_ids)
```

Comment:
active_model in context should be check . it is a 'purchase.order.line' or not.

and no need to declare variable Po_line. you can write statement:  .browse(active_ids).write(...)